### PR TITLE
feat(a11y): add landmarks

### PR DESCRIPTION
**Pull request by drxcheng**
first PR to improve https://github.com/Uniswap/interface/issues/3396

### CODE REVIEW

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -86,7 +85,7 @@ const HeaderElement = styled.div`
   `};
 `
 
-const HeaderLinks = styled(Row)`
+const HeaderLinks = styled.nav`
```
[the remark this answers is not in the document]
Comment (drxcheng) [reply src/components/Header/index.tsx:88]: Thank you for your consideration. I wasn't too sure about this part and I'd love to hear your feedback.

See this screenshot
Screen Shot 2022-03-02 at 9 19 19 PM

The only styling is no longer there (not overwritten by `HeaderLinks`) is `-webkit-box-pack` and `justify-content`, and it seems they have no impact here. I have tested Chrome and Firefox and they both look fine.

[the remark this answers is not in the document]
Comment (drxcheng) [reply src/components/Header/index.tsx]: Just pushed another commit. This should avoid any potential styling issues.

**Comment by drxcheng** [comment]
@zzmp

**Comment by vm** [comment]
closing as we have changed a good amount of the page structure since this pr was created

**Pull request closed without merging**